examples/client/Locomotion/ios/Locomotion.xcodeproj/project.pbxproj

```diff
@@ -72,9 +72,8 @@
 		ABC57D5443BA4AB7AE7EA145 /* Inter-ExtraBold.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = "Inter-ExtraBold.ttf"; path = "../src/assets/fonts/Inter-ExtraBold.ttf"; sourceTree = "<group>"; };
```

Comment:
Please align this file with master, CI Build is unable to find Google-Services and fails